addons/godot-xr-tools/user_settings/user_settings_ui.tscn

```diff
@@ -9,125 +9,106 @@ theme_override_font_sizes/font_size = 12
 script = ExtResource("1")
 
 [node name="Input" type="VBoxContainer" parent="."]
-anchors_preset = 15
-anchor_right = 1.0
-anchor_bottom = 1.0
-offset_top = 25.0
-grow_horizontal = 2
-grow_vertical = 2
+layout_mode = 2
 
 [node name="SnapTurning" type="HBoxContainer" parent="Input"]
-offset_right = 106.0
-offset_bottom = 24.0
+layout_mode = 2
 
 [node name="Label" type="Label" parent="Input/SnapTurning"]
-offset_top = 2.0
-offset_right = 78.0
-offset_bottom = 22.0
+layout_mode = 2
 theme_override_font_sizes/font_size = 12
 text = "Snap turning:"
 
 [node name="SnapTurningCB" type="CheckBox" parent="Input/SnapTurning"]
-offset_left = 82.0
-offset_right = 106.0
-offset_bottom = 24.0
+layout_mode = 2
+
+[node name="WebXR" type="HBoxContainer" parent="Input"]
```

Comment:
Same for the webXR UI items, I'd move them into a webXR tab

Reply:
Done in latest push!